autotest/gcore/basic_test.py

```diff
@@ -40,11 +40,13 @@
 # Nothing exciting here. Just trying to open non existing files,
 # or empty names, or files that are not valid datasets...
 
+non_existing_error_msg = 'No such file or directory'
```

Comment:
True. I got the impression that the autotest was only running on Linux (it still fails my mac in some ways, even trunk, whereas Travis passes). Are the autotests targeted to work across all platforms? If so, do you have suggestions on how to test on Windows?